doc/userguide/install.rst

```diff
@@ -364,9 +380,5 @@ Suricata packages.
 Advanced Installation
 ---------------------
 
-If you are using Ubuntu, you can follow
-:doc:`devguide/codebase/installation-from-git`.
-
-For other various installation guides for installing from GIT and for other operating
-systems, please check (bear in mind that those may be somewhat outdated):
+Various installation guides for installing from GIT and for other operating systems are maintained at:
```

Comment:
Ooooppps, we almost let a regression happen here. This was probably removed after a merge conflict.

Please, make sure we don't miss these recent changes added by https://github.com/OISF/suricata/commit/4ab4f711de1ad3424499f21a145f7f68d48c7173